drivers/display/display_ili9xxx.h

```diff
@@ -54,7 +54,18 @@
 /** Reset wait time (ms), ref 15.4 of ILI9XXX manual. */
 #define ILI9XXX_RESET_WAIT_TIME 5
 
+enum madctl_cmd_set {
+	CMD_SET_1,	/* Default for most of ili9xxx display controllers */
+	CMD_SET_2,	/* Used by ili9342c */
+};
+
+struct ili9xxx_quirks {
+	enum madctl_cmd_set orientation_cmd_set;
```

Comment:
enum madctl_cmd_set does not seem to be orientation specific, so why not just `enum madctl_cmd_set cmd_set;` ?